spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -310,6 +310,12 @@ object external {
     tblEntity.setAttribute("tableType", tableDefinition.tableType.name)
     tblEntity.setAttribute("createTime", new Date(tableDefinition.createTime))
     tblEntity.setAttribute("parameters", tableDefinition.properties.asJava)
+    tblEntity.setAttribute("schemaDesc", tableDefinition.schema.simpleString)
```

Comment:
Just FYI, the code will not conflict with #243, but given most of attributes being added here are not existing in Hive table model, either this or #243 should rebase and fix this once other side of patch gets merged.

Reply:
I guess we need to add the attributes only for the spark_ entities ?
For the hive_ entities assuming it gets created from HMS atlas hook should we deal with that here?

Reply:
> I guess we need to add the attributes only for the spark_ entities ?
I think "Yes".

> For the hive_ entities assuming it gets created from HMS atlas hook should we deal with that here?
These attributes are not existing in Hive models so we can just safely ignore these attributes.